w3-hw1-ThuyDung/cypress/constants/constants.js

```diff
@@ -0,0 +1,8 @@
+export const constants ={
+    //image
+    SYMBOL_INVALID: "url(\"data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='12' height='12' fill='none' stroke='%23dc3545' viewBox='0 0 12 12'%3e%3ccircle cx='6' cy='6' r='4.5'/%3e%3cpath stroke-linejoin='round' d='M5.8 3.6h.4L6 6.5z'/%3e%3ccircle cx='6' cy='8.2' r='.6' fill='%23dc3545' stroke='none'/%3e%3c/svg%3e\")",
```

Comment:
IDK how it works, but for E2E you can skip it, I think verify border color is enough